sample-code/src/orchestration.ts

```diff
@@ -145,6 +150,31 @@ export async function orchestrationTemplating(): Promise<OrchestrationResponse>
   });
 }
 
+/**
+ * Chat request to OpenAI through the Orchestration service using message history.
+ * @returns The orchestration service response.
+ */
+export async function orchestrationMessageHistory(): Promise<OrchestrationResponse> {
+  const orchestrationClient = (messages: TemplatingChatMessage) =>
+    new OrchestrationClient({
+      llm,
+      templating: {
+        template: messages
```

Comment:
[req] There is no need of an arrow function here. Create a client object with template { role: 'user', content: '{{?message}}' } and pass the question as inputParam. Use the same in both the calls. (The orchestration api for template is bad, but this is a cleaner way to demonstrate what we're doing.)